libs/logger/src/lib/logger.spec.ts

```diff
@@ -0,0 +1,46 @@
+import winston from 'winston';
+import { Logger, initializeLogger } from './logger';
+import { SerializeDevLog } from './serializers/serialize-dev-log';
+
+const production = !process.env.DEVELOP;
+
+const { transports } = winston;
+const { combine, timestamp, prettyPrint } = winston.format;
+const consoleTransport = new transports.Console();
+const w = winston.createLogger({
+  level: production ? 'info' : 'debug',
+  format: production
+    ? combine(timestamp(), prettyPrint({ colorize: true }))
+    : new SerializeDevLog(),
+  transports: [consoleTransport],
+  rejectionHandlers: [consoleTransport],
+  exceptionHandlers: [consoleTransport],
+});
+
+initializeLogger(w, production);
+
+Logger().info('Just some information');
```

Comment:
Not entirely sure what the purpose of all the logging statements achieve outside of the describe section.

Reply:
This was just for testing how everything looks. Because jest captures console log in strange way and does not actually shows how console looks i didn't used it there

I'll remove this file, i dont think we need it